packaging/language/bower.py

```diff
@@ -75,6 +80,10 @@
 
 description: Update packages based on bower.json to their latest version.
 - bower: path=/app/location state=latest
+
+description: install bower locally and run from there
+- npm: path=/app/location name=bower global=no
+- bower: path=/app/location localexec=node_modules/.bin
 '''
```

Comment:
Looks like the param was renamed to relative_execpath so the example needs to be changed as well.